data/strats.js

```diff
@@ -96,7 +96,7 @@ const strats = [
   {
     name: "ftm-fantohm",
     id: "ftm-fantohm-01",
```

Comment:
can you describe a case where you see two strats with the same name but diff ids? Maybe an upgrade of some sort?